helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/ticket/TicketService.java

```diff
@@ -106,4 +131,42 @@ protected TicketResponse buildResponse(Ticket ticket, SymUser agent, String mess
 
   protected abstract TicketResponse execute(Ticket ticket, SymUser agentUser);
 
+  protected void sendMessageWithShowHistoryFalse(Ticket ticket, Long agentId) {
```

Comment:
This method might be moved to AcceptTicketService subclass once only this component requires this behavior. Then, you should try to reuse the method to send message to client stream.

Reply:
Done.